plugins/rtmp-services/data/services.json

```diff
@@ -2510,19 +2510,29 @@
                 "max video bitrate": 20000
             }
         },
-        {
-            "name": "Bitmovin",
-            "more_info_link": "https://developer.bitmovin.com/docs/overview",
-            "stream_key_link": "https://bitmovin.com/dashboard/streams?streamsTab=LIVE",
-            "servers": [
-                {
-                    "name": "Streams Live",
-                    "url": "rtmp://live-input.bitmovin.com/streams"
-                }
-            ],
-            "recommended": {
-                "keyint": 2
-            }
-        }
+	{
+		"name": "Bitmovin",
+		"more_info_link": "https://developer.bitmovin.com/docs/overview",
+		"stream_key_link": "https://bitmovin.com/dashboard/streams?streamsTab=LIVE",
+		"servers": [
+			{
+				"name": "Streams Live",
+				"url": "rtmp://live-input.bitmovin.com/streams"
+			}
+		],
+		"recommended": {
+			"keyint": 2
+		}
+	},
+	{
+		"name": "MAVENX.GG",
+		"more_info_link": "https://mavenx.gg",
```

Comment:
Per our Services Submission Guidelines, `more_info_link` is meant to be a:
> Link that provides additional info about the service, presented in the UI as a button next to the services dropdown.

Linking to the homepage seems too vague/broad.